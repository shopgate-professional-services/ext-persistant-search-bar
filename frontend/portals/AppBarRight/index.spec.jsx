import React from 'react';
import { shallow } from 'enzyme';
import isIOSTheme from '@shopgate-ps/pwa-extension-kit/env/helpers/isIOSTheme';
import { UnwrappedAppBarRight as AppBarRight } from './index';

jest.mock('@shopgate/pwa-common/components/Swiper', () => () => (<div>Swiper</div>));
jest.mock('@shopgate-ps/pwa-extension-kit/env/helpers/isIOSTheme', () => () => true);

describe('<CouponField />', () => {
  it('should render as expected without any props', () => {
    const wrapper = shallow(<AppBarRight visible isIOSTheme={isIOSTheme} />);
    expect(wrapper).toMatchSnapshot();
  });
});
