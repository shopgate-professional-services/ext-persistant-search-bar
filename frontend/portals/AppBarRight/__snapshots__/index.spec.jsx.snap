// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CouponField /> should render as expected without any props 1`] = `""`;
